### ISSUE TITLE
Rename all resource, data source, and test files to be consistent

### ISSUE THREAD

**Issue by lafentres** [firehydrant/terraform-provider-firehydrant#35]
Not all of the files follow the same naming convention. 

Make sure every data source is named `<name>_data.go` and has a test file called `<name>_data_test.go`. There should be once data source per file.

Make sure every resource is named `<name>_resource.go` and has a test file called `<name>_resource_test.go`. There should be one resource per file.

**Comment by OneCricketeer** [comment]
Was there a reason the names were flipped?

> As a general convention, Terraform providers put each data source in their own file, named after the resource, **prefixed with `data_source_`**

https://learn.hashicorp.com/tutorials/terraform/provider-setup?in=terraform/providers

> As a general convention, Terraform providers put each resource in their own file, named after the resource, **prefixed with `resource_`**.

https://learn.hashicorp.com/tutorials/terraform/provider-create?in=terraform/providers

**Comment by lafentres** [comment]
Hey there @OneCricketeer! 

I'm not sure why the names were flipped when the provider was originally built. When I made this issue, the most important thing to me was that everything be named consistently. I considered changing everything to match the HashiCorp docs and providers I've worked on before (and that still might happen eventually) but felt it was too big at the time. So I thought getting the existing/future code consistent with itself was manageable and still helpful.

It is definitely the goal to eventually get everything in line with standard practices though.
